### ISSUE TITLE
Pop on screen notiffications get reset

### ISSUE THREAD

**Issue by mmjoshi** [Telegram-FOSS-Team/Telegram-FOSS#478]
I am using the latest version from F-Droid. I have switched off Pop on screen notifications for all messages. However, they get switched on automatically. No mater how many times I switch them off, they get switched on again. So irritating! How do I switch them off permanently?

I am using Nokia 6.1, Android 10 with so the latest secutity patches applied.

**Comment by Younes-L** [comment]
To make any permanent changes to telegram's notifications, do so using the settings inside of telegram app.
This applies to any version prior to 7.3, where some support for android's notification channels was introduced.
If you are referring to headsup notifications (at the top of the screen), you can disable them for each category (private chats, groups, channels) from telegram's settings by **decreasing the importance of the notifications**. By default the importance is set to "Urgent" , change it to "High" or lower
Related #369

> « you should not use android notification channels settings and [instead you should] manage all notifications setting within the app. App will overwrite everything you changed in system settings. » - developer of telegram (see https://t.me/tgbetachat/66523)

**Comment by mmjoshi** [comment]
Thanks for your tip.

Yes, I am talking about the headsup notifications at the top of the 
screen. I was changing the settings in Android to not show the pop up 
screen. Have changed the settings for Importance from Urgent to Medium. 
Will see if that works and revert.

**Comment by Younes-L** [comment]
fyi, Medium will also turn off notification sound

> Urgent —  plays sound, shows heads-up notification and icon in the status bar
High — plays sound, shows icon in the status bar, doesn't show heads-up notification 
Medium — shows icon in the status bar, doesn't play sound and doesn't show heads-up notification 
Low — doesn't show status bar icon, doesn't play sound and also doesn't show heads-up notification
via https://t.me/TelegramAndroidTalk/197238

**Comment by mmjoshi** [comment]
Thanks. One question. Will medium and high show notifications when the status bar is pulled down? Also what about the notification badges?

Sound is not important for me. The notification badges and notifications when the status bar is pulled down are. Of course the headsup notifications  should be turner off.

**Comment by Younes-L** [comment]
Yes, medium and high importance both show notification when the status bar is pulled down, and they don't affect notification dots.
As far as I know, android settings are ok to control notification dots (badges).
There's also a setting inside the telegram app for notifications badges, but ultimately I believe it's up to android system and the launcher app wether to show dots or not.

Feel free to join these groups for more information about telegram :
* https://t.me/telegramandroidtalk
* https://t.me/tgbetachat
